src/components/BaseAvatar.vue

```diff
@@ -32,14 +32,14 @@ watch(
     <img
       :src="
         imgUrl ||
-        `https://stamp.fyi/avatar/eth:${address}?s=${parseInt(size) * 2}`
+        `https://stamp.fyi/avatar/eth:${address}?s=${parseInt(size as string) * 2}`
```

Comment:
are you assuming size will always be there? if not, its better to put a default size

Reply:
Oh yeah adding default